ebu_tt_live/xsd1.1/ebutt_datatypes.xsd

```diff
@@ -65,7 +65,7 @@ Please note that the EBU-TT XML Schema is a helping document and NOT normative b
 	<xs:simpleType name="paddingType">
 		<xs:restriction base="xs:token">
 			<xs:pattern
-				value="(\d*\.?\d+(px|c|%))|([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))|([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))|([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))"
+				value="((\d*(\.\d+)?(px|c|%))|([+-]?\d*(\.\d+)?(px|c|%)))(\s((\d*(\.\d+)?(px|c|%))|([+-]?\d*(\.\d+)?(px|c|%))))*(\s((\d*(\.\d+)?(px|c|%))|([+-]?\d*(\.\d+)?(px|c|%))))*(\s((\d*(\.\d+)?(px|c|%))|([+-]?\d*(\.\d+)?(px|c|%))))*"
```

Comment:
oh sorry I see what you mean. But then since the `[+-]` has a `?` after it, surely the two cases one with it and one without are unnecessary duplicates?